### PR TITLE
Add mini-toolbar

### PR DESCRIPTION
**Pull request by Quorafind**
# I am submitting a new Community Plugin

## Repo URL

Link to my plugin: https://github.com/Quorafind/Obsidian-Mini-Toolbar

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [ ]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the tips in https://github.com/obsidianmd/obsidian-releases/blob/master/plugin-review.md and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

### CODE REVIEW

**Comment by Quorafind** [comment]
The original author has given me permission to maintain and publish this plugin. Thank you @aidenlx

**Comment by Quorafind** [comment]
@liamcain Sorry that I found you had reviewed it, I will make some changes to repo soon.

**Comment by joethei** [comment]
Hi @Quorafind,
I just wanted to confirm that you've seen Liams comment with the requested changes.
Do you still intend on submitting this plugin?

**Comment by ericaxu** [comment]
Hi there, to keep things tidy, we're going to temporarily close plugin PRs after a month of inactivity.

Feel free to tell us to re-open it when you're ready to continue. Thanks for your understanding!

**Comment by Quorafind** [comment]
> * [His original](https://github.com/Quorafind/Obsidian-Mini-Toolbar/blob/d1e4b34810e1832cb74966e7703803327a21458c/README.md#L44-L45) there's a typo in the readme, this should be a link, not an embed (it doesn't render)
> * [state.selection.ranges[0].from](https://github.com/Quorafind/Obsidian-Mini-Toolbar/blob/d1e4b34810e1832cb74966e7703803327a21458c/src/modules/toolbar.ts#L39) style nit: you can reference sel here.
> * [start: anchor > head ? head : anchor](https://github.com/Quorafind/Obsidian-Mini-Toolbar/blob/d1e4b34810e1832cb74966e7703803327a21458c/src/modules/toolbar.ts#L43) you could do something like `let [start, end] = [anchor, head].sort()`
> * [app](https://github.com/Quorafind/Obsidian-Mini-Toolbar/blob/d1e4b34810e1832cb74966e7703803327a21458c/src/modules/defaultCommand.ts#L22) It's highly discouraged that you use the global app instance in your plugin. It's currently accessible from the global context for debugging purposes, and might be removed in the future. Instead, you should use the reference to App that you have from your plugin instance.
> * [import { around } from "monkey-around";](https://github.com/Quorafind/Obsidian-Mini-Toolbar/blob/d1e4b34810e1832cb74966e7703803327a21458c/src/popper/view-plugin.ts#L10-L11) unused.

Hi @liamcain , I have fixed all you mention. https://github.com/Quorafind/Obsidian-Mini-Toolbar/commit/d68c82f3fef2b2e63029de250412bd04b541eda3

File: community-plugins.json [hunk 1]
```diff
@@ -7635,5 +7635,13 @@
...
     "description": "Improved comment toggling.",
     "repo": "MrGVSV/obsidian-better-comment-toggle"
+  },
+  {
+     "id": "mini-toolbar",
+     "name": "Mini Toolbar",
+     "author": "AidenLx & Boninall",
+     "description": "Mini context toolbar in editor for Obsidian.",
+     "repo": "quorafind/obsidian-mini-toolbar",
+     "branch": "main"
```
Comment (liamcain) [inline community-plugins.json:7645]:
```suggestion
     "repo": "quorafind/obsidian-mini-toolbar"
```

**Pull request merged**